csrc/scheduler/greedy.cpp

```diff
@@ -749,9 +786,26 @@ std::unordered_map<TensorView*, TensorView*> partitionFusion(
 
   std::unordered_map<TensorView*, TensorView*> tv_to_constrained_tv_map;
 
-  // Register self mappings for constrained tensors
+  // Register self and sibling mappings for constrained tensors
```

Comment:
Not related to this PR.

For things like nvfp4 quantization, where we have two outputs with mismatch extent for the constrainted id, I think it makes sense to still partition them together. I don't know if TransformPropagator would be able to replay it though, I imagine we cannot map two ID with different extent, even though we are assigning them with the same parallelization type?!

Reply:
btw, cc'ing @protonu for the conversation we were having regarding nvfp4 quant kernel scheduling.